src/iterative/KOSMAPOSL/KOSMAPOSLReconstruction.cxx

```diff
@@ -617,8 +628,8 @@ estimate_stand_dev_for_anatomical_image()
 
                     for (int x=min_x;x<= max_x;x++)
                     {
-                        if((*anatomical_prior_sptr)[z][y][x]>=0 && (*anatomical_prior_sptr)[z][y][x]<=1000000){
-                        kmean += (*anatomical_prior_sptr)[z][y][x];
+                        if((*anatomical_prior_sptr[i])[z][y][x]>=0 && (*anatomical_prior_sptr[i])[z][y][x]<=1000000){
```

Comment:
is it documented anywhere that you have a hard-wired threshold of 1000000? Rather scary... Please create an issue for it such that we can sort this out in a separate PR